### PR TITLE
Rename ready -> preparing.

### PR DESCRIPTION
**Pull request by hjoliver**
Easiest part of the task state rationalization (just a one-to-one renaming)

These changes partially address #4013 

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] Appropriate change log entry included.
- [x] (master branch) covered by doc issue https://github.com/cylc/cylc-doc/issues/163
- [x] No dependency changes.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
One line diff in `tests/f/graphql/01-workflow.t`

File: cylc/flow/tui/__init__.py [hunk 1]
```diff
@@ -75,7 +75,7 @@
     f'{TASK_STATUS_WAITING}': '\u25cb',
 
     # TODO: remove with https://github.com/cylc/cylc-admin/pull/47
```
Comment (oliver-sanders) [inline cylc/flow/tui/__init__.py:77]:
```suggestion
```

**Comment by hjoliver** [comment]
Tests fixed.

**Comment by hjoliver** [comment]
@kinow this is the cylc-flow sibling of https://github.com/cylc/cylc-ui/pull/564

**Comment by kinow** [comment]
> @kinow this is the cylc-flow sibling of [cylc/cylc-ui#564](https://github.com/cylc/cylc-ui/pull/564)

Oh, missed this one, thanks @hjoliver !

**Review by kinow — APPROVED** [review]

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**